openbas-model/src/main/java/io/openbas/database/model/ExecutionTraces.java

```diff
@@ -135,12 +138,13 @@ public ExecutionTraces(
       List<String> identifiers,
       String message,
       ExecutionTraceAction action,
-      Agent agent) {
+      Agent agent,
+      Instant time) {
     this.injectStatus = injectStatus;
     this.status = status;
     this.identifiers = identifiers == null ? new String[0] : identifiers.toArray(new String[0]);
     this.message = message;
-    this.time = Instant.now();
+    this.time = time == null ? Instant.now() : time;
```

Comment:
I'm not sure defaulting to Instant.now here is the right way to go. It's a bit of an hidden logic. Up for discussion, clearly :)